app/models/package_manager/npm.rb

```diff
@@ -58,7 +58,18 @@ def self.deprecation_info(db_project)
         .last
 
       message = last_stable_version&.dig("deprecated")
-      is_deprecated = !message.nil?
+      # sometimes message is a boolean `false`, and it may be `true` sometimes (not sure but
+      # may as well handle it, apparently we return any type we like)
+      if message.nil?
+        is_deprecated = false
+      elsif [true, false].include?(message)
```

Comment:
originally I wrote this as `is_a? Boolean` but Ruby doesn't actually have a boolean class, only TrueClass and FalseClass.